### DISCUSSION TITLE
Sound problems

### DISCUSSION THREAD

**Discussion by ramzidaher** [google/model-viewer#4411] [Q&A]
I have been working with model viewer for over a month or so, I have added animation and sound. It was working fine on all devices but all of a sudden on specific devices especially google phones it's not playing(model viewer not loading properly) it would show the model but the title and the link will not show. I got so confused can someone help, is it an issue from Google or is it from my side? Even though I rolled back to the old code that I knew was working 100%, it's not working now Please help me! Thank you.

**Comment by elalish** [comment]
Impossible to know without a URL where we can repro. And be specific about what problem device/browser. Also, your title says "sound problems", but your description says "model viewer not loading properly" - those are very different things. Which is it (since model-viewer does not itself provide sound)?

**Reply by ramzidaher** [reply]
Hi there yes sorry I didn't include the code, but after a bunch of testing, it still didn't work. But what is happening mainly is that on Android the sound doesn't always play and the model viewer functionality like title and link does not show, but the model shows, it seems that the model viewer isn't working properly. 

This is the glitch project
https://glitch.com/edit/#!/funny-protective-gigantoraptor

**Reply by ramzidaher** [reply]
Also while testing I noticed that it doesn't always open ARCore app, sometimes it asks and sometimes it doesn't and I think that's the main problem.

**Reply by elalish** [reply]
Agreed. You can restrict to WebXR mode (where browser things like sound can work) with `ar-modes="webxr quick-look"`. Then you'll see it failing to open at all and you can check your JS console for messages on why.
